NuGet.config

```diff
@@ -3,5 +3,6 @@
   <packageSources>
     <add key="NuGet" value="https://api.nuget.org/v3/index.json" />
     <add key="AspNetCore" value="https://dotnet.myget.org/F/aspnetcore-ci-release/api/v3/index.json" />
+    <add key="ANCM" value="https://dotnet.myget.org/F/aspnetcoremodule/api/v3/index.json" />
```

Comment:
Because there aren't dev and release versions of it, and we didn't want to push it to all the feeds.